content/efcore/getting-started/defining-unique-identifiers.md

```diff
@@ -0,0 +1,62 @@
+# Defining Unique Identifiers 
+ 
+In this lesson we will learn about unique identifiers, how they are configured by default in EF Core, and how to manually configure them when we don't follow the convention. 
+ 
+## Primary Keys 
+ 
+When defining our data model, we need to ensure that each entity class includes a property that can be used to uniquely identify instances of that entity. These unique identifiers are referred to as **primary keys**, and they map to the concept of primary keys in a relational database.  
+ 
+By convention, EF Core assumes that a property named `Id` or `<type name>Id` is the primary key for an entity. For example, in our `Book` class below, the property named `Id` is assumed to be the primary key. 
+ 
+```{.snippet}  
+public class Book 
+{ 
+    public int Id { get; set; }  // You could also use 'BookId' here 
+    public string Title { get; set; } 
```

Comment:
That way, we can also just share the same `Book` implementation for each of the unique identifier code examples.